modules/operator/scripts/install_helm.template.sh

```diff
@@ -4,16 +4,7 @@
 
 echo 'installing helm'
 
-wget https://get.helm.sh/helm-v${helm_version}-linux-amd64.tar.gz 2> /dev/null
-
-
-tar zxvf helm-v${helm_version}-linux-amd64.tar.gz 2> /dev/null
-
-sudo mv linux-amd64/helm /usr/local/bin 2> /dev/null
-
-rm -f helm-v${helm_version}-linux-amd64.tar.gz 2> /dev/null
-
-rm -rf linux-amd64 2> /dev/null
+sudo yum install -y helm-${helm_version} 2> /dev/null
```

Comment:
cool. I think I'll nick this for the OKE scripts. How do I check which versions of helm are available?

Reply:
`yum list --show-duplicates helm`